proto/metadata.proto

```diff
@@ -386,7 +386,9 @@ message Metadata {
     bool isis_metric_style_telemetry_unsupported = 134;
     // Devices do not support configuring Interface-ref under Static-Route Next-Hop
     bool static_route_next_hop_interface_ref_unsupported = 135;
-
+    // Devices doesnt support acl tunnel encapsulation configuration.
+    bool tunnel_acl_encapsulation_config_unsupported = 136;
```

Comment:
Please add bug ID for new deviation: unnel_acl_encapsulation_config_unsupported

Reply:
Thanks for the review. Added // Juniper : b/318985874 now